packages/mgt-components/README.md

```diff
@@ -55,6 +55,40 @@ The components can be used on their own, but they are at their best when they ar
     <mgt-agenda group-by-day></mgt-agenda>
     ```
 
+
+## <a id="disambiguation">Disambiguation</a>
+
+MGT is built using [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components). Web components use their tag name as a unique key when registering within a browser. Any attempt to register a component using a previously registered tag name results in an error being thrown when calling `CustomElementRegistry.define()`. In scenarios where multiple custom applications can be loaded into a single page this created issues for MGT, most notably in developing custom SharePoint web parts.
```

Comment:
> custom SharePoint web parts

Let's generalize it to "developing solutions using SharePoint Framework." This issue applies to all UI elements that you can build with SPFx, such as web parts, extensions, field customizers, etc.